src/app/components/header/userMenu/userMenu.component.html

```diff
@@ -11,6 +11,6 @@
   </button>
 </mat-menu>
 
-<button mat-button [matMenuTriggerFor]="userMenu">
```

Comment:
The same as I said before about span inside a button

Reply:
Got rid of this and several others